charts/zeebe-benchmark/Chart.yaml

```diff
@@ -10,7 +10,7 @@ sources:
 dependencies:
   - name: camunda-platform
     repository: "https://helm.camunda.io"
-    version: "13.0.0-alpha8"
+    version: "13.1.1"
```

Comment:
:question: is this compatible with version < 8.8? 
Since we are using now unified config stuff probably not? 
Not sure if there's a way to not break it.